Testreglar/1.1.1/App/app-1.1.1f.json

```diff
@@ -0,0 +1,129 @@
+{
+	"namn": "App-1.1.1f Tidsbaserte medier har tekstalternativ 2023",
+	"id": "app-1.1.1f",
+	"testlabId": 509,
+	"versjon": "1.0",
+	"type": "App",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>Ikke-tekstlig innhold som er tidsbaserte medier, har kort tekstalternativ med en beskrivende identifikasjon av det tidsbaserte mediet.</p>",
```

Comment:
```suggestion
	"kravTilSamsvar": "<p>Video og lydklipp har et tilgjengelig navn med en beskrivende identifikasjon.</p>",
```

Reply:
I spørsmål 3.2 spør vi om video eller lyd er direktesendt. Dersom vi tolker at kravet til tekstalternativ (1.1.1) bare gjelder for forhåndsinnspilt innhold må det går frem i kravet til samsvar.